### PR TITLE
Nzpmc 002/implement welcome page

### PR DESCRIPTION
**Pull request by ruby-oujo**
**Issue**

Updated Homepage with Vuetify components 

**Discussion** 

To do: 
- Video needs to be updated with the NZPMC video 
- "This is a welcome page" will be removed 

**Risk**

None 

**Reviewers**

@Will-ZYS  @dtor068

### CODE REVIEW

File: src/views/Homepage.vue [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+}
+li {
+    display: inline-block;
+    margin: 5px 5px;
+    margin-bottom: 20px;
+}
+a {
+    color: #d0e0d8;
+}
+</style>
```
Comment (scorpionknifes) [inline src/views/Homepage.vue:129]: Is this file still used?

**Review by scorpionknifes — CHANGES_REQUESTED** [review]
Looks great, just one change!

**Review by Will-ZYS — APPROVED** [review]
LGTM

**Review by scorpionknifes — APPROVED** [review]
LGTM

**Review by matthewaptaylor — APPROVED** [review]
LGTM

**Pull request merged**